ezgui/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Dustin Carlino <dabreegster@gmail.com>"]
 edition = "2018"
 
 [features]
-default = ["glium-backend"]
+default = ["glow-backend"]
```

Comment:
You also need to change `game/Cargo.toml` if your intention is to make this the default everywhere; it specifies `glium` right now. Feel free to do that in a followup (which I'm guessing will delete glium?)